tljh/configurer.py

```diff
@@ -191,6 +203,43 @@ def update_traefik_api(c, config):
     c.TraefikTomlProxy.traefik_api_password = config['traefik_api']['password']
 
 
+def set_cull_idle_service(c, config):
+    """
+    Set Idle Culler service
+    """
+    cull_cmd = [
+       sys.executable, '/srv/src/tljh/cull_idle_servers.py'
+    ]
+    if config['services']['cull']['timeout']:
```

Comment:
Since you are accessing `config['services']['cull']` a lot, it would make this more readable to pull it out into its own variable.

```python
cull_config = config['services']['cull']
```